test/files/17-patches/expected-apply-output

```diff
@@ -0,0 +1,43 @@
+
+Working on file:/etc/symlink
+  store at target/var/lib/holo/files/base/etc/symlink
+     patch target/usr/share/holo/files/17-patches/etc/symlink.patch
+
+patching symbolic link symlink
+
+Working on file:/etc/symlink-to-plain
+  store at target/var/lib/holo/files/base/etc/symlink-to-plain
+     patch target/usr/share/holo/files/17-patches/etc/symlink-to-plain.patch
+
+patching symbolic link symlink-to-plain
+patching file symlink-to-plain
+
+Working on file:/etc/txtfile
+  store at target/var/lib/holo/files/base/etc/txtfile
+     patch target/usr/share/holo/files/17-patches/etc/txtfile.patch
+
+patching file txtfile
+
+Working on file:/etc/txtfile-to-symlink
+  store at target/var/lib/holo/files/base/etc/txtfile-to-symlink
+     patch target/usr/share/holo/files/17-patches/etc/txtfile-to-symlink.patch
+
+patching file txtfile-to-symlink
+patching symbolic link txtfile-to-symlink
+
+Working on file:/etc/txtfile-with-fuzz
+  store at target/var/lib/holo/files/base/etc/txtfile-with-fuzz
+     patch target/usr/share/holo/files/17-patches/etc/txtfile-with-fuzz.patch
+
+patching file txtfile-with-fuzz
+Hunk #1 succeeded at 1 with fuzz 1.
+
+Working on file:/etc/txtfile-with-garbage
+  store at target/var/lib/holo/files/base/etc/txtfile-with-garbage
+     patch target/usr/share/holo/files/17-patches/etc/txtfile-with-garbage.patch
+
+patching file txtfile-with-garbage
+patching file garbage
+patching file ls
```

Comment:
I don't know how hard it would be to implement, but to avoid confusion and/or surprises, I'd rather have the apply step fail if some parts of the patch are not applicable to the target file.